From b6bed8d4e9453bc74ba021c8c17e20c3b5964c37 Mon Sep 17 00:00:00 2001
From: Alex Suykov <alex.suykov@gmail.com>
Date: Thu, 21 Jan 2016 22:00:34 +0100
Subject: [PATCH] Add missing header include for ssize_t

The code uses ssize_t which is usually defined in <unistd.h>.

Signed-off-by: Alex Suykov <alex.suykov@gmail.com>
---
 host/lib/fmap.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/host/lib/fmap.c b/host/lib/fmap.c
index c95338d..e3db826 100644
--- a/host/lib/fmap.c
+++ b/host/lib/fmap.c
@@ -6,6 +6,7 @@
 
 #include <stdio.h>
 #include <string.h>
+#include <unistd.h>
 
 #include "fmap.h"
 
-- 
2.6.4

